starknet_devnet/blueprints/rpc/structures/payloads.py

```diff
@@ -456,6 +467,37 @@ def make_declare(declare_transaction: RpcBroadcastedDeclareTxn) -> DeprecatedDec
     return declare_tx
 
 
+def make_declare_v2(declare_transaction: RpcBroadcastedDeclareTxnV2) -> Declare:
+    """Convert RpcBroadcastedDeclareTxnV2 to Declare"""
+    nonce = declare_transaction.get("nonce")
+
+    contract_class = declare_transaction["contract_class"]
+
+    if "abi" not in contract_class:
+        contract_class["abi"] = ""
```

Comment:
I made it this way, because in starkware's ContractClass model, it's not optional:
```python
@marshmallow_dataclass.dataclass(frozen=True)
class ContractClass(ValidatedMarshmallowDataclass):
    """
    Represents a contract class in the StarkNet network.
    """

    contract_class_version: str
    sierra_program: List[int] = field(metadata=fields.felt_as_hex_list_metadata)
    entry_points_by_type: Dict[EntryPointType, List[ContractEntryPoint]]
    abi: str
```